numba/cuda/cudadrv/nvvm.py

```diff
@@ -127,13 +132,34 @@ def __new__(cls):
 
         return cls.__INSTANCE
 
+    def __init__(self):
+        ir_versions = self.get_ir_version()
+        self._majorIR = ir_versions[0]
+        self._minorIR = ir_versions[1]
+        self._majorDbg = ir_versions[2]
+        self._minorDbg = ir_versions[3]
+
+    @property
+    def is_nvvm70(self):
+        return (self._majorIR, self._minorIR) >= (1, 6)
```

Comment:
Note: https://docs.nvidia.com/cuda/libnvvm-api/group__compilation.html#group__compilation_1g0c22d2b9be033c165bc37b16f3ed75c6
Says that a buffer should contain NVVM IR version 1.6 in a LLVM 7.x representation (text or bitcode), hence where this comes from.

Reply:
Will add a note in the code.

Reply:
thanks